_data/validator-libraries-modern.yml

```diff
@@ -1,24 +1,18 @@
 - name: .NET
   anchor-name: dotnet
   implementations:
-    - name: Json.NET Schema
-      url: https://www.newtonsoft.com/jsonschema
-      date-draft: [2019-09]
-      draft: [7, 6, 4, 3]
-      license: "AGPL-3.0-only"
     - name: JsonSchema.Net
       url: https://github.com/gregsdennis/json-everything
       date-draft: [2020-12, 2019-09]
       draft: [7, 6]
       license: MIT
+    - name: Json.NET Schema
+      url: https://www.newtonsoft.com/jsonschema
+      date-draft: [2019-09]
+      draft: [7, 6, 4, 3]
+      license: "AGPL-3.0-only"
 - name: C
-  implementations:
-    - name: WJElement
-      url: https://github.com/netmail-open/wjelement
-      date-draft:
-      draft: [4, 3]
-      license: LGPL-3.0
-      notes: "Draft-06+ progress: issue [17](https://github.com/netmail-open/wjelement/issues/17#issuecomment-390899432)"
+  notes: All known implementations are now obsolete.
```

Comment:
Perhaps "No known implementations support draft-06 or later" since casual readers won't know what "obsolete" means, probably not even if we document it somewhere.